src/geode/mesh/core/solid_mesh.cpp

```diff
@@ -298,7 +299,9 @@ namespace
             }
         }
         OPENGEODE_EXCEPTION( safety_count < MAX_SAFETY_COUNT,
-            "[SolidMesh::compute_polyhedra_around_vertex] Too many polyhedra "
+            "[SolidMesh::compute_polyhedra_around_vertex] Solid: ",
+            solid.name(),
+            " Too many polyhedra "

```

Comment:
try to homogenize the log messages: here in this file you have the version with the separator"-" and without